test/unit/job_test.rb

```diff
@@ -1273,11 +1273,11 @@
       end
 
       it "duplicate jobs are not loaded" do
-        out = Sidekiq::Cron::Job.load_from_hash @jobs_hash
+        out = Sidekiq::Cron::Job.load_from_hash! @jobs_hash
         assert_equal out.size, 0, "should have no errors"
         assert_equal Sidekiq::Cron::Job.all.size, 2, "Should have 2 jobs after load"
 
-        out_2 = Sidekiq::Cron::Job.load_from_hash @jobs_hash
+        out_2 = Sidekiq::Cron::Job.load_from_hash! @jobs_hash
```

Comment:
I was referring the `ScheduleLoader` tests: https://github.com/sidekiq-cron/sidekiq-cron/blob/master/test/unit/schedule_loader_test.rb

Reply:
Yes, seemed like the right thing to do. Have updated the tests.